connector/email/src/main/java/it/pagopa/selfcare/mscore/connector/email/EmailConnectorImpl.java

```diff
@@ -70,5 +78,17 @@ public void sendMail(String templateName, List<String> destinationMail, File pdf
         }
         log.trace("sendMessage end");
     }
+    public byte[] zipBytes(String filename, File pdf) throws IOException {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ZipOutputStream zos = new ZipOutputStream(baos);
+        ZipEntry entry = new ZipEntry(filename);
+        byte[] pdfToByte = FileCopyUtils.copyToByteArray(pdf);
+        zos.putNextEntry(entry);
+        zos.write(pdfToByte);
+        zos.closeEntry();
+        zos.close();
+        return baos.toByteArray();
+    }
+
```

Comment:
you might use try-catch for closing stream when there are exceptions...for example:
try ( ByteArrayOutputStream baos = new ByteArrayOutputStream();
        ZipOutputStream zos = new ZipOutputStream(baos);) {

} catch(IOException e) {
  ....
}

look at https://javarevisited.blogspot.com/2014/10/right-way-to-close-inputstream-file-resource-in-java.html#axzz8D5mU58K7